src/main/java/io/stargate/sgv2/jsonapi/service/operation/tables/WhereCQLClauseAnalyzer.java

```diff
@@ -239,6 +241,39 @@ private void checkFilteringOnComplexColumns(Map<CqlIdentifier, TableFilter> iden
     }
   }
 
+  /**
+   * Check if any non $eq filter is used.
+   *
+   * <p>e.g. Table UpdateOne and DeleteOne, DATA API does not know the exact modified row count in
+   * DB, so it needs to restrict on the filters. Make sure only $eq is used for DeleteOne and
+   * UpdateOne.
+   */
+  private void checkOnlyEqUsage(Map<CqlIdentifier, TableFilter> identifierToFilter) {
```

Comment:
from discussions: we should not allow a sort() clause for updateOne and deleteOne for Tables because CQL does not allow sorting on these commands. Create a new ticket if we need to do the work.

Reply:
confirmed, these sort() on deleteOne and updateOne have already been covered by

- UNSUPPORTED_SORT_FOR_TABLE_UPDATE_COMMAND
- UNSUPPORTED_SORT_FOR_TABLE_DELETE_COMMAND